frontend/facebookServiceMock.js

```diff
@@ -0,0 +1,9 @@
+// facebookService.js mock functions
+
+// returns 'Username' by default and 'Home' if the first argument is "1"
+const loginWithFacebookMock = (home) => {
```

Comment:
remove mock from all of these functions and i don't believe we can control inserting the argument through this way...do it like how i sent you